dlrs/README.md

```diff
@@ -10,9 +10,8 @@ The Deep Learning Reference Stack, is an integrated, highly-performant open sour
 
 This open source community release is part of an effort to ensure AI developers have easy access to all features and functionality of Intel platforms.To offer more flexibility, there are multiple versions of the Deep Learning Reference Stack:
 
-* Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI)  
-* TensorFlow with Eigen optimized for Intel Architecture  
-* TensorFlow with Intel® Deep Neural Networks Library (Intel® DNNL) primitives
+* Tensorflow 2.0 and 1.15 with Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI) 
+* TensorFlow with Eigen optimized for Intel Architecture
 * OSS PyTorch DLRS Docker image with Eigen
 * PyTorch DLRS Docker image with Intel® Math Kernel Library and Intel® Deep Neural Networks Library (Intel® DNNL) 
```

Comment:
- [Fliar -  A very simple framework for state-of-the-art Natural Language Processing](https://github.com/zalandoresearch/flair)
- [PyTorch-lightning - PyTorch wrapper for ML researchers](https://github.com/williamFalcon/pytorch-lightning)

Reply:
Better have these on releasenotes.md?

Reply:
+1